ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/textures.cpp

```diff
@@ -79,8 +79,10 @@ unsigned char* graphics_copy_texture_pixels(int texture, unsigned* fullwidth, un
 
   unsigned char* ret = new unsigned char[((*fullwidth)*(*fullheight)*4)];
   glPixelStorei(GL_PACK_ALIGNMENT, 1);
+  #if !defined(__ANDROID__) || defined(glGetTexImage)
```

Comment:
This is the entirety of the logic for this file. You can't just snip out the beating heart of a single function because your phone doesn't support it. If this function is no longer common due to ES, extract it to its own header/source.